packages/alfa-rules/src/common/predicate/is-marked-decorative.ts

```diff
@@ -1,16 +1,32 @@
-import {Role} from "@siteimprove/alfa-aria";
-import {Element} from "@siteimprove/alfa-dom";
+import { Role } from "@siteimprove/alfa-aria";
+import { Element } from "@siteimprove/alfa-dom";
+import { Predicate } from "@siteimprove/alfa-predicate";
 
 /**
- * @see https://github.com/Siteimprove/sanshikan/blob/master/glossary/marked-as-decorative.md
- * Check if an element is marked as decorative by looking at its role but
- * without conflict resolution. If the result is "none" or "presentation", then
- * the element is marked as decorative.
+ * Check if an element is marked as decorative.
  */
-export function isMarkedDecorative(element: Element): boolean {
-  return (
-    Role.from(element, { allowPresentational: true })
-      // Element is marked as decorative if at least one browser thinks so.
-      .some((r) => r.some(Role.hasName("none", "presentation")))
+export const isMarkedDecorative: Predicate<Element> = (element) => {
```

Comment:
The Sanshikan definition https://github.com/Siteimprove/sanshikan/blob/master/glossary/marked-as-decorative.md is also considering `<a>` without `href` to be marked as decorative (and possibly a few other cases). Which this is not doing anymore.

Reply:
Yep, we need to change the definition in Sanshikan to align with the updated ACT-R definition.